cli/tests/unit/write_file_test.ts

```diff
@@ -256,8 +256,8 @@ Deno.test(
       assert(e instanceof Error);
       assertEquals(e.name, "AbortError");
     }
-    const stat = Deno.statSync(filename);
-    assertEquals(stat.size, 0);
+    // it's racy to check the file length, so just check that the file exists
+    assertExists(filename);
```

Comment:
This isn't guaranteed to happen either I think. This may be flaky.

Reply:
Yeah, I removed these checks. What's asserted now is probably good enough.